Amplify/DevMenu/View/LogEntryRow.swift

```diff
@@ -5,10 +5,10 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#if canImport(UIKit)
```

Comment:
I presume this is gated because of `LogEntryItem`? Since `LogEntryItem` is just a model without any `UIKit` specific features, does it make sense to drop this `#if canImport(UIKit)` check as well as the one in `LogEntryItem`?

Reply:
Ah, I see further down that this is due to the `UIViewRepresentable` `SearchBar`. Disregard my previous comment.